### PR TITLE
Member profile of Abhijeet Gaurav is added

### LINKED ISSUE #4937: [Community] Member Profile: Abhijeet Gaurav
**Current Behavior**

@abhijeetgauravm has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @abhijeetgauravm as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @abhijeetgauravm
- Twitter: [@abhijeetgauravm](https://twitter.com/abhijeetgauravm)
- LinkedIn: [abhijeet-gaurav-971b01a4](abhijeet-gaurav-971b01a4)
- Meshery Cloud: 
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by VivekBaldha0712**
**Description**

This PR fixes #4937

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 43c540fb6d0bf9e78f78a2c7b75e0855ec62b078 at: https://650d7fe4993b9643606a2623--layer5.netlify.app

File: src/collections/members/abhijeet-gaurav/Abhijeet_Gaurav.jpeg
Comment (abhijeetgauravm) [inline src/collections/members/abhijeet-gaurav/Abhijeet_Gaurav.jpeg:1]: hey @VivekBaldha0712 lets use .webp format. Can u do the required change?

**Comment by VivekBaldha0712** [comment]
Sure

**Comment by leecalcote** [comment]
Yay, @abhijeetgauravm! 🎈

File: src/collections/members/abhijeet-gaurav/index.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+name: Abhijeet Gaurav
+position: Contributor
+image_path: ./Abhijeet_Gaurav.jpeg
+github: abhijeetgauravm
+twitter: abhijeetgauravm
+linkedin: abhijeet-gaurav-971b01a4
+layer5: c9432f12-f9d2-4c99-8704-8b750e42c023
+location: Patna, India
+bio: "Hello! I'm Abhijeet, a tech enthusiast on a journey of discovery. I'm constantly exploring the ever-evolving landscape of DevOps, Open Source and Web Development. I'm eager to contribute my energy and fresh perspectives to the field. I firmly believe that learning is a lifelong adventure, and I'm here to soak up knowledge like a sponge. Let's connect, share ideas, and embark on this exciting journey of growth and discovery together! 🚀🌍"
+status: Active
```
Comment (leecalcote) [inline src/collections/members/abhijeet-gaurav/index.mdx:11]:
```suggestion
badges: community
status: Active
```

**Review by hirentimbadiya — APPROVED** [review]
@VivekBaldha0712, LGTM 👍🏻

**Review by abhijeetgauravm — APPROVED** [review]
LGTM! Thanks for the contribution @VivekBaldha0712

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Yay!

**Pull request merged**